tests/x11/pidgin/pidgin_IRC.pm

```diff
@@ -51,11 +51,14 @@ sub run {
     }
 
     # CTCP Version and warning about scan may appear
-    assert_screen([qw(pidgin-ctcp-version pidgin-irc-sledtesting)]);
+    assert_screen([qw(pidgin-ctcp-version pidgin-ready)]);
     if (match_has_tag('pidgin-ctcp-version')) {
         wait_screen_change { send_key "ctrl-w"; }    # close it
     }
 
+    # Assure that pidgin is ready
+    assert_screen("pidgin-ready");
```

Comment:
As you are already looking for "pidgin-ready" in before I guess you can just move that assert_screen into the above if to not do the same check twice in the case that 'pidgin-ctcp-version' does *not* show up

Reply:
Moved.. new verification run: http://10.100.12.105/tests/335